src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,23 @@
-export const TodoInfo = () => {};
+import classNames from 'classnames';
+
+import { UserInfo } from '../UserInfo';
+import { Todos } from '../../types/Todos';
+
+type Props = {
+  todo: Todos;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  return (
+    <article
+      data-id={todo.id}
+      className={classNames('TodoInfo', {
+        'TodoInfo--completed': todo.completed,
+      })}
+    >
+      <h2 className="TodoInfo__title">{todo.title}</h2>
+
+      <UserInfo user={todo.user} />
```

Comment:
The `UserInfo` component is used here with the assumption that `todo.user` is always defined. If `todo.user` can be `null` or `undefined`, consider adding a conditional check to handle these cases to prevent potential runtime errors.